modules/hitl2/src/views/full/Components/Sidebar.tsx

```diff
@@ -1,35 +1,31 @@
+import { Collapsible, lang } from 'botpress/shared'
 import _ from 'lodash'
-import React, { FC, useContext, useEffect, useState } from 'react'
-import cx from 'classnames'
+import React, { FC, Fragment, useContext, useEffect, useState } from 'react'
 
+import { EscalationType } from '../../../types'
 import { Context } from '../app/Store'
+import style from '../style.scss'
 
 import { ApiType } from './../Api'
-import { EscalationType } from '../../../types'
-
-import { lang } from 'botpress/shared'
-import Collapsible from '../../../../../../src/bp/ui-shared-lite/Collapsible'
-import UserProfile from './UserProfile'
-import CommentList from './CommentList'
+import Comment from './Comment'
 import CommentForm from './CommentForm'
-
-import styles from './../style.scss'
+import UserProfile from './UserProfile'
 
 interface Props {
   api: ApiType
   escalation: EscalationType
 }
 
-const Sidebar: FC<Props> = props => {
-  const { api } = props
-
+const Sidebar: FC<Props> = ({ escalation, api }) => {
+  const { id, comments, userConversation } = escalation
   const { dispatch } = useContext(Context)
 
   const [expanded, setExpanded] = useState(false)
 
-  async function createComment(content: string) {
+  const createComment = async (content: string) => {
```

Comment:
Curious - why do you change the way this function is declared?

Reply:
just for consistency throughout the app, no real reason